trace-sdk/build.gradle

```diff
@@ -9,6 +10,7 @@ android {
         targetSdkVersion 30
         versionCode = "$project.versionCode" as int
         versionName "$project.version"
+        buildConfigField('String', 'VERSION_NAME', "\"${project.version}\"")
```

Comment:
this has to be added to get access to the VERSION_NAME in the NetworkClient - as per https://issuetracker.google.com/issues/158695880 :(